### DISCUSSION TITLE
This is bug.

### DISCUSSION THREAD

**Discussion by LTonJin** [apache/skywalking#8587] [General]
### This is bug.
Collect Metrics Manually
Use the setPerformance method to report metrics at the moment of page loaded or any other moment meaningful.

Set the SDK configuration item autoTracePerf to false to turn off automatic reporting performance metrics and wait for manual triggering of escalation.
Call ClientMonitor.setPerformance(object) method to report
Examples
```
import ClientMonitor from 'skywalking-client-js';

ClientMonitor.register({
  collector: 'http://127.0.0.1:8080',
  service: 'browser-app',
  serviceVersion: '1.0.0',
  pagePath: location.href,
  useFmp: true,
  + autoTrancePerf: false
});

ClientMonitor.setPerformance({
  collector: 'http://127.0.0.1:8080',
  service: 'browser-app',
  serviceVersion: '1.0.0',
  pagePath: location.href,
  useFmp: true,
  + autoTrancePerf: true
});
// If you call both functions at the same time, you will not get window.performance data,This is due to setTimeout.
// if register function param autoTrancePerf is false,that setPerformance function param autoTrancePerf  must is true.Otherwise we can't get window.performance data.

```

**Comment by wu-sheng** [comment]
Please make your issue title and contents in good format and meaning. This is a bug is not a good practice.
